components/tableNodes/NodeTable.component.tsx

```diff
@@ -94,11 +94,13 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
                   {}
                 </TableCell>
                 <TableCell component="th" scope="row">
-                  {channel.amount_msat + " msats"}
+                  {channel.amount_msat + "s"}
                 </TableCell>
                 <TableCell component="th" scope="row">
-                  {intoSatoshi(Number(btcPrice), Number(channel.amount_msat)) +
-                    " USD"}
+                  {intoSatoshi(
+                    Number(btcPrice),
+                    parseInt(channel.amount_msat)
```

Comment:
```suggestion
                    parseInt(channel.amount_msat)
```

good catch, but core lightning is migrating to number, so this should be already a number. We should make sure that this will not crash if `amount_msat` is a number